docs/cudf/source/user_guide/pandas-comparison.md

```diff
@@ -0,0 +1,155 @@
+# Comparison of cuDF and Pandas
```

Comment:
This page is missing from the `user_guide/index.md` so I don't see it in the rendered docs?

Reply:
Added.